src/request_queue.js

```diff
@@ -600,6 +611,7 @@ export class RequestQueueLocal {
         ]);
 
         this.pendingCount = pending.length;
+        this._handledCount = handled.length;
```

Comment:
I know, but there already is a function called `handledCount`, so I needed something else and this seemed like the most viable option. And it's called `handledCount` because there already was a function by this name in `RequestList`...

Got any idea for a property name without `_` that would be in line with the `pendingCount` ?